kong.conf.default

```diff
@@ -328,72 +328,48 @@
-                                 # other nodes in the cluster will stop
-                                 # attempting to connect to it.
-                                 # Recommended to be at least `60`.
-
-#cluster_profile = wan           # The timing profile for inter-cluster pings
-                                 # and timeouts. If a `lan` or `local` profile
-                                 # is used over the Internet, a high rate of
-                                 # failures is risked as the timing contraints
-                                 # would be too tight.
-                                 # Accepted values are `local`, `lan`, `wan`.
+# This section allows for configuring the behavior of Kong regarding the
+# caching of such configuration entities.
+
+#db_update_frequency = 5         # Frequency (in seconds) at which to check for
+                                 # updated entities with the datastore.
+                                 # When a node creates, updates, or deletes an
+                                 # entity via the Admin API, other nodes need
+                                 # to wait for the next poll (configured by
+                                 # this value) to eventually purge the old
+                                 # cached entity and start using the new one.
+
+#db_update_propagation = 0       # Time (in seconds) taken for an entity in the
+                                 # datastore to be propagated to replica nodes
+                                 # of another datacenter.
+                                 # When in a distributed environment such as
+                                 # a multi-datacenter Cassandra cluster, this
+                                 # value should be the maximum number of
+                                 # seconds taken by Cassandra to propagate a
+                                 # row to other datacenters.
+                                 # When set, this property will increase the
+                                 # time taken by Kong to propagate the change
+                                 # of an entity.
+                                 # Single-datacenter setups or PostgreSQL
+                                 # servers should suffer no such delays, and
+                                 # this value can be safely set to 0.
+
+#db_cache_ttl = 3600             # Time-to-live (in seconds) of an entity from
+                                 # the datastore when cached by this node.
+                                 # If set to 0, such cached entities never
+                                 # expire.
+
+#db_cache_miss_ttl = 300         # Time-to-live (in seconds) of a miss from the
+                                 # datastore. Such misses can be cached for a
+                                 # different period of time that hits
+                                 # (configurable via `cache_ttl`).
```

Comment:
Both of those TTL values are a safeguard against eventually stale data. It's a very bad idea imo to not have a TTL in our cache, and to solely rely on the invalidation events being called. It could go wrong, somehow...